.github/workflows/sycl-precommit.yml

```diff
@@ -69,27 +65,38 @@ jobs:
           else
             echo 'arc_tests="Matrix/"' >> "$GITHUB_OUTPUT"
           fi
-  test:
-    needs: [build, detect_changes, determine_arc_tests]
-    if: ${{ always() && !cancelled() && needs.build.outputs.build_conclusion == 'success' }}
+
+  test_linux:
+    needs: [build_linux, detect_changes, determine_arc_tests]
+    if: ${{ always() && !cancelled() && needs.build_linux.outputs.build_conclusion == 'success' }}
     strategy:
       fail-fast: false
       matrix:
         include:
-          - name: AMD/HIP
+          # XXX CUDA testing is in a separate workflow because it runs on AWS
+          # which has different access restrictions/requirements. See
+          # sycl-linux-precommit.yml if you're looking for the CUDA precommit
+          # definition
+          - name: E2E on AMD/HIP
             runner: '["Linux", "amdgpu"]'
             image: ghcr.io/intel/llvm/ubuntu2204_build:latest-0300ac924620a51f76c4929794637b82790f12ab
             image_options: -u 1001 --device=/dev/dri --device=/dev/kfd
             target_devices: ext_oneapi_hip:gpu
-          - name: Intel
+
+          - name: E2E on Intel GEN12; level zero GPU, OpenCL CPU
```

Comment:
Everywhere in the code: `Level Zero` - both words should start with capital letters